index.html

```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
                 </div>
             </div>
         </section>
+        <section class = "sect_content">
+            <div class = "container">
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Timer.png" alt="timer" class="feature__icon">
+                        </div class="feature__icon_container">
+                    <h6>Designer guff</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Dashboard.png" alt="dashboard" class="feature__icon">
+                        </div>
+                    <h6>Dashboard</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                    <div class="feature__icon_container">
+                        <img src="./img/Stats.png" alt="stats" class="feature__icon">
+                    </div>
+                    <h6>Detailed stats</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                        <div class="feature__icon_container_little">

```

Comment:
re-indent please

Reply:
`Prettier` and `Eslint` have different sights of intendation of ```@font-face``` block in CSS, so I turned Prettier off till I will find a decision of this issue